src/cljs/rems/application.cljs

```diff
@@ -758,7 +758,7 @@
               [:th (text :t.form/comment)]
               [:th (text :t.form/date)]]]
             (into [:tbody]
-                  (for [e (sort-by :time > events)]
+                  (for [e (reverse events)]
```

Comment:
Shouldn't time be sent accurately and formatted in UI inaccurately?

Reply:
It is. The rounding to millisecond accuracy happens in rems.application/application-header and is used to avoid showing to the user duplicate notifications about comment requests, for which there are one event per receiver but the applicant doesn't see who the receiver is, so it will show up confusingly as duplicates in the event list.

And anyways sorting by timestamp is fundamentally incorrect. It's normal to have multiple events produced at the same millisecond, especially when a single command produces multiple events. The database currently sorts the events by ID and those IDs are produced using a `serial` column (i.e. `nextval` from a sequence), so there can be gaps in the ID values and they might not be committed in monotonically increasing order, but for our purposes it's probably good enough. We should however lock the application being updated, because currently if the same application is updated concurrently, it could produce an inconsistent sequence of events.